cli/aws_orbit/remote_files/cdk/foundation.py

```diff
@@ -93,6 +96,23 @@ def __init__(self, scope: Construct, id: str, context: "FoundationContext", **kw
 
         self.user_pool: cognito.UserPool = self._create_user_pool()
 
+        # self.artifacts_domain = codeartifact.CfnDomain(
```

Comment:
we should eliminate commented code

Reply:
I thought when we bump the cdk versions, we would just uncomment the codeartifact construct here. Should i remove them totally?